pathlib2/__init__.py

```diff
@@ -87,9 +87,8 @@ def _py2_fsencode(part):
         # getfilesystemencoding can return None, in that
         # case fall back to ascii
         return part.encode(sys.getfilesystemencoding() or 'ascii')
-    else:
```

Comment:
@yezz123 did you do these with an automated tool? If so I'd be interested in using it for some of my other projects

Reply:
No Manually, i just configure my IDLE and this can help